release.sh

```diff
@@ -6,33 +6,60 @@
 # state.  The second phase is modifying files, tagging, commiting and pushing
 # to pypi
 
+OFFICIAL_GIT_REPO='git@github.com:taskcluster/taskcluster-client.py'
+
+# step into directory containing this script
+cd "$(dirname "${0}")"
+
+# exit in case of bad exit code
 set -e
 
 # Phase 1: Testing things
-if [ -z $VERSION ] ; then
-  echo "You must specify a version using \$VERSION"
-  exit 1
+
+# VERSION should have form x.y.z where x, y, z are positive integers
+if ! echo "${VERSION}" | grep -q '^\(0\|[1-9][0-9]*\)\.\(0\|[1-9][0-9]*\)\.\(0\|[1-9][0-9]*\)$'; then
```

Comment:
i'm not a regex expert, but doesn't `0\|[1-9][0-9]*` equate to `[0-9]+`?  My understanding is that the pipe is an or operator that works on the next token, in this case [1-9].  If they're both [0-9], then using + with one instead of * would acheive the same result.  I'm probably missing something here, it's been a while since i've had to write a big regex